migrations/20200606105847-convert-collective-long-description-to-html.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import Promise from 'bluebird';
+import showdown from 'showdown';
+
+const converter = new showdown.Converter();
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const collectives = await queryInterface.sequelize.query(
+      `
+        SELECT * FROM "Collectives"
```

Comment:
In production, this query matches 6000+ collectives. Selecting the fields that we want will save some precious RAM:

```suggestion
        SELECT "id", "longDescription" FROM "Collectives"
```